packages/server/src/index.ts

```diff
@@ -18,9 +18,8 @@ async function loadData() {
 }
 await loadData();
 
-
-// update cache every hour
-const job = await new CronJob('0 * * * *', updateCacheWithRemote);
+// fetch remotely every minute
```

Comment:
0 * * * *  is hourly

Reply:
true!

Reply:
[07ca979](https://github.com/VandeurenGlenn/blue_server/pull/20/commits/07ca979b9a840456813c84aa917b905dcd599848)